content/en/docs/concepts/services-networking/endpointslice.md

```diff
@@ -0,0 +1,97 @@
+---
+reviewers:
+- freehan
+title: Endpoint Slice
+feature:
+  title: Endpoint Slices
+  description: >
+    Scalable tracking of network endpoints in a Kubernetes cluster.
+
+content_template: templates/concept
+weight: 10
+---
+
+
+{{% capture overview %}}
+
+{{< glossary_definition term_id="endpointslice" length="short" >}}
+
+Endpoint Slices provide a simple way to track network endpoints within a
+Kubernetes cluster. Currently in alpha, they are intended to eventually replace
```

Comment:
As per the style guide, [avoid making statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future)

Reply:
Fixed, thanks!